content/posts/ai_konferens_2024-02-07.md

```diff
@@ -0,0 +1,59 @@
+title: Summering av eventet AI i klimatets tjänst onsdagen 7 februari 2024
+date: 2024-02-08
```

Comment:
2024-02-07 to make it match the date of the title.

Reply:
done